### ISSUE TITLE
Works with any type of video?

### ISSUE THREAD

**Issue by Ruke805** [PatchyVideo/MMD-Translator#2]
The proposal of the project is very cool, I was wondering if it is possible to work with any kind of video files, mkv, mp4, avi? 
I'm looking for a tool to make a subtitle by hard subs, making via OCR, and save as subtitle file, maybe your program could help

**Comment by zyddnys** [comment]
Yes, it works for all types of videos so long as its supported by OpenCV's video decoder
